src/main/java/com/java_professional/diledsys/homework/dayWeek/codeDay/DayOfWeek.java

```diff
@@ -0,0 +1,60 @@
+package com.java_professional.diledsys.homework.dayWeek.codeDay;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class DayOfWeek {
+    private static int past = 0;
+    private static String range = "";
+    private static String sortString="";
+
+    public static String converter(String str) {
+        str = sortirovka(str);
+        StringBuilder sb = new StringBuilder();
+        for (String s : str.split("")) {
+            if (Integer.parseInt(s)>7|| Integer.parseInt(s)<1){
+                System.out.println(s+" вне заданного диапазона");
+                try {
+                    throw new Exception();
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
```

Comment:
```suggestion
                    throw new Exception();
```
If you need to generate some exception, throw it to upper level.
Also add an error message to exception constructor and use logger